.github/workflows/main.yaml

```diff
@@ -45,15 +45,15 @@ jobs:
     strategy:
       matrix:
         ide-version:
-          - clion:2024.2
-          - goland:2024.2
-          - ideaIC:2024.2
-          - ideaIU:2024.2
-          - phpstorm:2024.2
-          - pycharmPC:2024.2
-          - pycharmPY:2024.2
-          - riderRD:2024.2
-          - webstorm:2024.2
+          - clion:2024.3
```

Comment:
Not for this PR, but would there be value in testing a least one variant with our minimum version?

Reply:
Good idea!